src/Uno.UWP/Globalization/Calendar.cs

```diff
@@ -12,65 +13,59 @@ public partial class Calendar
-				case global::System.Globalization.GregorianCalendar _: return CalendarIdentifiers.Gregorian;
-				case global::System.Globalization.HebrewCalendar _: return CalendarIdentifiers.Hebrew;
-				case global::System.Globalization.HijriCalendar _: return CalendarIdentifiers.Hijri;
-				case global::System.Globalization.JapaneseCalendar _: return CalendarIdentifiers.Japanese;
-				case global::System.Globalization.KoreanCalendar _: return CalendarIdentifiers.Korean;
-				case global::System.Globalization.TaiwanCalendar _: return CalendarIdentifiers.Taiwan;
-				case global::System.Globalization.ThaiBuddhistCalendar _: return CalendarIdentifiers.Thai;
-				case global::System.Globalization.UmAlQuraCalendar _: return CalendarIdentifiers.UmAlQura;
-				case global::System.Globalization.PersianCalendar _: return CalendarIdentifiers.Persian;
-				case global::System.Globalization.ChineseLunisolarCalendar _: return CalendarIdentifiers.ChineseLunar;
-
+				global::System.Globalization.JulianCalendar _ => CalendarIdentifiers.Julian,
+				global::System.Globalization.GregorianCalendar _ => CalendarIdentifiers.Gregorian,
+				global::System.Globalization.HebrewCalendar _ => CalendarIdentifiers.Hebrew,
+				global::System.Globalization.HijriCalendar _ => CalendarIdentifiers.Hijri,
+				global::System.Globalization.JapaneseCalendar _ => CalendarIdentifiers.Japanese,
+				global::System.Globalization.KoreanCalendar _ => CalendarIdentifiers.Korean,
+				global::System.Globalization.TaiwanCalendar _ => CalendarIdentifiers.Taiwan,
+				global::System.Globalization.ThaiBuddhistCalendar _ => CalendarIdentifiers.Thai,
+				global::System.Globalization.UmAlQuraCalendar _ => CalendarIdentifiers.UmAlQura,
+				global::System.Globalization.PersianCalendar _ => CalendarIdentifiers.Persian,
+				global::System.Globalization.ChineseLunisolarCalendar _ => CalendarIdentifiers.ChineseLunar,
 				// Not supported by UWP as of 2019-05-23
 				// https://docs.microsoft.com/en-us/uwp/api/windows.globalization.calendaridentifiers
 				// case CalendarIdentifiers.VietnameseLunar: return new global::System.Globalization.VietnameseLunarCalendar();
 				// case CalendarIdentifiers.TaiwanLunar: return new global::System.Globalization.TaiwanLunarCalendar();
 				// case CalendarIdentifiers.KoreanLunar: return new global::System.Globalization.KoreanLunarCalendar();
 				// case CalendarIdentifiers.JapaneseLunar: return new global::System.Globalization.JapaneseLunarCalendar();
-
-				default: throw new ArgumentException(nameof(calendar));
-			}
+				_ => throw new ArgumentException(nameof(calendar)),
+			};
 		}
 
-		private static string GetDefaultClock()
-		{
+		private static string GetDefaultClock() =>
 			// Windows.System.UserProfile.GlobalizationPreferences.Clocks.FirstOrDefault();
-			return new DateTime(1983, 9, 9, 13, 0, 0).ToString("g", CultureInfo.CurrentCulture).Contains("PM")
+			new DateTime(1983, 9, 9, 13, 0, 0).ToString("g", CultureInfo.CurrentCulture).Contains("PM")
```

Comment:
Codacy found an issue: [Change this call to 'new DateTime(1983, 9, 9, 13, 0, 0).ToString("g", CultureInfo.CurrentCulture).Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6178917)